source/Handlebars/Configuration/Compatibility.cs

```diff
@@ -6,9 +6,10 @@
     public class Compatibility
     {
         /// <summary>
-        /// If <see langword="true"/> enables support for <c>@last</c> in object properties iterations. Not supported in Handlebarsjs.
+        /// If <see langword="true"/> enables support for <c>@last</c> in object properties iterations.
         /// </summary>
-        public bool SupportLastInObjectIterations { get; set; } = false;
+        [System.Obsolete]
```

Comment:
Add message explaining why it's obsolete and what people should do with it (e.g., remove all usages)